pkgs/tools/networking/miniupnpd/default.nix

```diff
@@ -15,22 +16,40 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-b9cBn5Nv+IxB58gi9G8QtRvXLWZZePZYZIPedbMMNr8=";
   };
 
-  buildInputs = [ iptables-legacy libuuid openssl ];
+  buildInputs = [ iptables-legacy libuuid openssl ]
+    ++ lib.optionals (firewall == "nftables") [ libmnl libnftnl ];
   nativeBuildInputs= [ pkg-config makeWrapper ];
 
 
   # ./configure is not a standard configure file, errors with:
   # Option not recognized : --prefix=
   dontAddPrefix = true;
 
+  FW = firewall;
```

Comment:
Please prefix all environment variables `env.FW = ...`, `env.IPV6 = ...` or just do `env = { FW = ...; … }`
(this is a way to separate options from env vars we pass to the drv environment)

Reply:
I use configureFlags now